internal/common/config/configer.go

```diff
@@ -114,8 +176,18 @@ func processAgentFlags(cfg *AgentConfig) error {
 	return nil
 }
 
-var processEnv = func(config *Config) error {
-	err := env.Parse(config)
+var processEnv = func(config *ServerConfig) error {
```

Comment:
Please exclude the `processEnv` variable

Reply:
done